### ISSUE TITLE
Problem on other phones ?

### ISSUE THREAD

**Issue by theayryan** [commonsguy/cwac-camera#268]
Is there a problem in the camera running on some  phones ? I tried on HTC one e8 and an one plus one. Works fine on the HTC on the one plus one it shows the preview but app stops as soon as I press capture But video recording works fine.

Any idea why ?

**Comment by commonsguy** [comment]
> Any idea why ? 

No. You did not provide me with a stack trace. You also did not provide me with a description of how to reproduce the problem, including code if this is not reproducible via the demo app. Without those things, there is nothing that I can really do.
